spec/deps/git.go

```diff
@@ -75,13 +75,13 @@ func (gs *Git) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
-// Name returns the repository in a go-like format (github.com/user/repo/subdir)
+// Name returns the repository in a go-like format (example.com/user/repo/subdir)
 func (gs *Git) Name() string {
 	return fmt.Sprintf("%s/%s/%s%s", gs.Host, gs.User, gs.Repo, gs.Subdir)
 }
 
 // LegacyName returns the last element of the packages path
-// example: github.com/ksonnet/ksonnet-lib/ksonnet.beta.4 becomes ksonnet.beta.4
+// example: example.com/ksonnet/ksonnet-lib/ksonnet.beta.4 becomes ksonnet.beta.4
```

Comment:
```suggestion
// example: github.com/ksonnet/ksonnet-lib/ksonnet.beta.4 becomes ksonnet.beta.4
```

This is actually located there, no need to change it